### DISCUSSION TITLE
Androïd - Dropin Cancelation

### DISCUSSION THREAD

**Discussion by ljourdan** [Adyen/adyen-react-native#284] [Q&A]
We need to catch the cancelation of the payment on Android when the customer Cancel the payment on the Payment Method selection.
Right now, the didFail is not called and we don't know how to catch this event properly.
What is the best way to proceed? (we have the same issue to tackle when the user click on the return button but we know how to catch that).

**Comment by descorp** [comment]
Hey @ljourdan 

This feature is not yet available on our Native Android SDK.
We have #112 for this, will keep you posted.

**Comment by ljourdan** [comment]
Thx for your quick answer.
Any idea when this could be tackled?

**Reply by descorp** [reply]
[Android v5](https://github.com/Adyen/adyen-android/releases/tag/5.0.0-beta01) is in BETA stage now.
We are looking to adopt and provide some early ALPHA/BETA `2.0.0` for React Native before the end of the year.
